client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/buffer/BufferPacker.java

```diff
@@ -81,16 +89,35 @@ public void process(Buffer buffer, int subIdx) throws InterruptedException {
                 buffer.asByteBuf(),
                 BufferUtils.HEADER_LENGTH_PREFIX,
                 buffer.readableBytes() - BufferUtils.HEADER_LENGTH_PREFIX);
+        logger.debug(
+            "cachedBuffer pack partition:{} type:{}, length:{}",
+            currentSubIdx,
+            buffer.getDataType(),
+            buffer.getSize(),
+            buffer.readableBytes() - BufferUtils.HEADER_LENGTH_PREFIX);
+
         buffer.recycleBuffer();
       } else {
         Buffer dumpedBuffer = cachedBuffer;
         cachedBuffer = buffer;
+        logger.debug(
```

Comment:
can we implements one method to reduce code for same logging pattern.

Reply:
create function to do this